pytorch_lightning/trainer/progress.py

```diff
@@ -153,6 +153,10 @@ def load_state_dict(self, state_dict: dict) -> None:
         self.total.load_state_dict(state_dict["total"])
         self.current.load_state_dict(state_dict["current"])
 
+    def reset_on_restart(self):
+        self.total.reset_on_restart()
```

Comment:
what does it mean to reset the total counter? I don't understand this

Reply:
Good catch ! This is a bug.